mirage/helpers.ts

```diff
@@ -26,7 +26,6 @@ export function registerNode(
     node.contributors.models.forEach(
         contributor => server.create('contributor', { node: registration, users: contributor.users }),
     );
-    return registration;
```

Comment:
This doesn't seem like something that should be in your PR. Can you put this back?

Reply:
Of course, thank you.